frontend/src/test/java/com/java/academy/gym/controller/InternationalizationControllerTest.java

```diff
@@ -0,0 +1,63 @@
+package com.java.academy.gym.controller;
+
+import com.java.academy.gym.dto.LocaleUserMessagesRequestDto;
+import com.java.academy.gym.service.InternationalizationService;
+import com.java.academy.gym.util.JsonUtil;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+public class InternationalizationControllerTest {
+
+    private MockMvc mockMvc;
+
+    @Mock
+    private InternationalizationService mockInternationalizationService;
+
+    @InjectMocks
+    private InternationalizationController internationalizationController;
+
+    @Before
+    public void init(){
+        MockitoAnnotations.initMocks(this);
+        mockMvc = MockMvcBuilders
+                .standaloneSetup(internationalizationController)
+                .build();
+    }
+
+    @Test
+    public void shouldReturnMapOfUserMessages() throws Exception {
+        Map<String, String> userMessages = new HashMap<>();
+        userMessages.put("MK1", "Message1");
+        userMessages.put("MK2", "Message2");
+        when(mockInternationalizationService.findUserMessagesByKeysAndLanguageCode(any(), any()))
```

Comment:
I think it's better to specify what messages exactly you'll be passing inside. So instead of any() pass on concrete values.